pkgs/tools/misc/online-judge-verify-helper/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonApplication
+, colorlog
+, fetchFromGitHub
+, git
+, importlab
+, online-judge-tools
+, pyyaml
+, setuptools
+, toml
+}:
+
+buildPythonApplication rec {
+  pname = "online-judge-verify-helper";
+  version = "5.6.0";
+
+  src = fetchFromGitHub {
+    owner = "online-judge-tools";
+    repo = "verification-helper";
+    rev = "v${version}";
+    sha256 = "sha256-sBR9/rf8vpDRbRD8HO2VNmxVckXPmPjUih7ogLRFaW8=";
+  };
+
+  patches = [ ./fix-paths.patch ];
+  postPatch = ''
+    substituteInPlace onlinejudge_verify/main.py --subst-var-by git ${git}
```

Comment:
did you mean?
```suggestion
    substituteInPlace onlinejudge_verify/main.py --subst-var-by git ${git}/bin/git
```

Reply:
`/bin/git` is appended here:
https://github.com/NixOS/nixpkgs/pull/157294/files#diff-4fcb2c4716deaeefd540e70d71ad27b802daf485c5cd20250f4a808bab1f5673R11-R12

Reply:
Why not wrap the binary and add git to PATH instead? that is less patching and easier to update